bats/tests/helpers/profile.bash

```diff
@@ -225,7 +225,13 @@ remove_profile_entry() {
         profile_plutil -remove "${setting%.}" || return
         ;;
     linux)
-        profile_jq "del(.${setting%.})" || return
+        profile_jq "
+            if has(.${setting%.}) then
```

Comment:
Unfortunately `has` takes a key name as the parameter, and not a path. So `has("foo")` works, but `has("foo.bar")` does not and needs to be `.foo|has("bar")` instead.

I wonder if there is a simpler way.

Reply:
Looks like we want `try` instead of `has`:

```console
$ echo '{"foo": {"bar": 42}}' | jq 'if try(.foo.bar) then del(.foo.bar) else error("no") end'; echo $?
{
  "foo": {}
}
0

$ echo '{"foo": {"bar": 42}}' | jq 'if try(.foo.baz) then del(.foo.bar) else error("no") end'; echo $?
jq: error (at <stdin>:1): no
5
```